src/index.html

```diff
@@ -17,6 +29,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="header_logo"
+        href="#"
+      >
+        <img
+          class="header_img"
+          src="./images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link is-active first"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav_item">
+            <a
+              class="nav_link"
+              href="#"
+            >
```

Comment:
It will be better if you fix code style everywhere if you have only 2 attributes
```suggestion
            <a class="nav_link" href="#">
```

Reply:
Коли я роблю наново пункти з 13 по 18 в мене атрибути самі перескакують на новий рядок. Навіть коли я роблю ось так
 <a class="nav_link" href="#">

Reply:
<a class="nav_link" href="#"  >